README.md

```diff
@@ -8,23 +8,24 @@ This is the officially supported Python library for using Adyen's APIs.
+| [Balance Platform API](https://docs.adyen.com/api-explorer/balanceplatform/2/overview)            | The Balance Platform API enables you to create a platform where you can onboard your users as account holders and create balance accounts, cards, and business accounts.                                                                                                                                                                                                                                                                                                | balancePlatform       | **v2**            |
+| [Checkout API](https://docs.adyen.com/api-explorer/Checkout/71/overview)                          | Our latest integration for accepting online payments.                                                                                                                                                                                                                                                                                                                                                                                                                   | checkout              | **v71**           |
+| [Data Protection API](https://docs.adyen.com/development-resources/data-protection-api)           | Endpoint for requesting data erasure.                                                                                                                                                                                                                                                                                                                                                                                                                                   | dataProtection        | **v1**            |
+| [Legal Entity Management API](https://docs.adyen.com/api-explorer/legalentity/3/overview)         | Endpoint to manage legal entities                                                                                                                                                                                                                                                                                                                                                                                                                                       | legalEntityManagement | **v4**            |
```

Comment:
We could also improve the description maybe? From API Explorer `The Legal Entity Management API enables you to manage legal entities that contain information required for verification.`